### DISCUSSION TITLE
Using parcels with a constant velocity field

### DISCUSSION THREAD

**Discussion by mathildejutras** [Parcels-code/Parcels#1179] [General]
Hello,
I am looking for an easy way to launch an experiment with one velocity field, that does not change with time. Is there a way to do that?
Thank you.

**Comment by erikvansebille** [comment]
Hi @mathildejutras, thanks for your question. This should work out of the box, by creating a `FieldSet` with just one netcdf file as input. If you get an `OutofTime` Error, you can add `allow_time_extrapolation=True` to the creation of the Fieldset
